README.md

```diff
@@ -6,7 +6,7 @@
 <a href='https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA'><img src='https://img.shields.io/static/v1?label=chat&logo=slack&message=Slack&color=brightgreen' alt='Slack'  align="center"/></a>
 <a href="https://codecov.io/gh/modin-project/modin"><img src="https://codecov.io/gh/modin-project/modin/branch/master/graph/badge.svg" align="center"/></a>
 <a href="https://github.com/modin-project/modin/actions"><img src="https://github.com/modin-project/modin/workflows/master/badge.svg" align="center"></a>
-<a href="https://modin.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=latest" align="center"></a>
+<a href="https://modin.readthedocs.io/en/0.13.2/?badge=0.13.2"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=0.13.2" align="center"></a>
```

Comment:
Should we fixate PyPI link for the point release too?

Reply:
Ideally yes, but I didn't find a way in fury badges to specify _any_ version - it automatically finds latest and shows it.
I can obviously change the link without changing the badge itself, but I find it confusing to show one version but link to another.

Reply:
Maybe we should revert this PR #3591?